freeipa/src/main/java/com/sequenceiq/freeipa/api/model/users/UsersyncEndpoint.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.freeipa.api.model.users;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import com.sequenceiq.freeipa.api.model.ContentType;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Path("/usersync")
+@Consumes(MediaType.APPLICATION_JSON)
+@Api(value = "/usersync", description = "Synchronize users to FreeIPA", protocols = "http,https")
```

Comment:
I think FreeIPA will be bound to a particular environment so we are thinking of including it in the API as path parameter. In that case this would need to have an env param also. 

@lacikaaa

Reply:
We've agreed on not having env params in the path.